napari/layers/base/base.py

```diff
@@ -173,8 +173,6 @@ class Layer(KeymapProvider, MousemapProvider, ABC):
         coordinates of each layer. For multiscale data the coordinates are in
         the space of the currently viewed data level, not the highest resolution
         level.
-    position : tuple
-        Cursor position in world coordinates.
```

Comment:
What with coordinates property? where it is updated?

Reply:
Sorry, I shouldn't have added these removals to this PR, which is more like a hotfix. I should probably just put them back in and create a separate PR.

In the napari codebase, the `position` attribute is set in the callback I removed. It used to be a deprecated property along with `coordinates`, which is not defined anywhere in code now. At some point those public properties were removed (not sure which PR did it), but these vestiges remained.